lib/collectionspace/converter/extension/commission.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
```

Comment:
@Paulmulonzia - This is an extension, so I put it in its own module.

Reply:
got it